deploy/kicbase/Dockerfile

```diff
@@ -21,6 +21,8 @@
 # for a kubernetes node image, it doesn't contain much we don't need
 FROM ubuntu:focal-20200925
 
+ARG BUILDKIT_VERSION="v0.7.2"
```

Comment:
nit: I think it would help readability to run this line right before we install buildkit, so it's easy to see everything at once